Support installation of help topics handled by mercurial-common.install.

--- a/setup.py
+++ b/setup.py
@@ -280,8 +280,7 @@
         extmodules.append(inotify)
         packages.extend(['hgext.inotify', 'hgext.inotify.linux'])
 
-packagedata = {'mercurial': ['locale/*/LC_MESSAGES/hg.mo',
-                             'help/*.txt']}
+packagedata = {'mercurial': ['locale/*/LC_MESSAGES/hg.mo']}
 
 def ordinarypath(p):
     return p and p[0] != '.' and p[-1] != '~'
--- a/mercurial/help.py
+++ b/mercurial/help.py
@@ -73,9 +73,11 @@
             module = __file__
         base = os.path.dirname(module)
 
+        docdir = '/usr/share/mercurial/help'
         for dir in ('.', '..'):
-            docdir = os.path.join(base, dir, 'help')
-            if os.path.isdir(docdir):
+            tmpdocdir = os.path.join(base, dir, 'help')
+            if os.path.isdir(tmpdocdir):
+                docdir = tmpdocdir
                 break
 
         path = os.path.join(docdir, topic + ".txt")
